File: iniciante/javascript/1065.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let countEntradas = 0
let countPares = 0

while(countEntradas++ < 5){
    let entrada = parseInt(lines.shift())
    if(entrada % 2 ===0){
        countPares++
    }
}

console.log(`${countPares} valores pares`)

File: iniciante/python/1065.py
# -*- coding: utf-8 -*-

count = 0
entrada = []
count_par = 0

while count < 5:
    entrada.append(int(input()))
    if entrada[count] % 2 == 0:
        count_par += 1
    count += 1

print(f'{count_par} valores pares')
